code/style.css

```diff
@@ -1,4 +1,201 @@
+*, *::before, *::after { box-sizing: inherit;}
+
 html {
-  background: #0025ff;
-  color: #fff;
+    display: flex;
+    background: #7e7171; 
+    color: #fff;
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+  }
+    header { 
+        display: flex;
+        background-color: black;
+        border: 5px outset red;
+        align-items: center;
+        justify-content: space-between;
+        border-radius: 10px;
+        width: 100%;
+    }
+
+    .logo {
+        display: flex;
+        border: 10px solid white;
+        background-color: white;
+        height: 100%;
+        align-items: center;
+        justify-content:center;
+        flex-direction: column;
+    }
+
+    .header-child {
+        margin: 5px;
+        width: 20%;
+        height: 80%;
+    }
+
+    .titel {
+        display: flex;
+        font-size: 100%;
+        text-align: center;
+        width: 100%;
+        height:100%;
+        color: black;
+        font-size: initial;
+    }
+
+    .navigation{
+          display: none;
+    }
+
+      .technico {
+            background-color: #d89191;
+            text-align: center;
+            height:  100%;
+            width: 100%; 
+      }
+
+      .zebra {
+          height:100%;
+          width: 100%;
+          border: 5px solid white;
+       }
+
+      .small-news {
+        border:2px solid transparent;
+        display: grid;
+        width:auto;
+        grid-template-columns: repeat(1, 1fr);
+        align-content: center;
+        justify-content: center;
+        gap: 4px;
+      }
+
+      img {
+        display: flex;
+        border: solid black;
+        width: 100%;
+        align-content: center;
+        justify-content: center;
+      }
+
+      .small-news-1 {
+              border: solid blue;
+      }
+
+      .small-news-2 {
+              border: solid white;
+      }
+
+      .small-news-3 {
+             border: solid green;
+      }
+
+      .small-news-4 {
+  border: solid red;
+      }
+
+      .big-news-tekst {
+        display: flex;
+        justify-content: center;
+        text-align: center;
+      }
+
+@media (min-width: 768px) {
+  .small-news {
+    grid-template-columns: 1fr 1fr;
+  }
+  .big-news {
+    display: flex;
+    border: 2px solid transparent;
+  }
+
+  .navigation {
+    background-color: white;
+    box-shadow: 0px 0px 10px var(--clr-gray100);
+    margin: 0;
+    padding:1rem 0;
+    border-radius:var(--radius);
+    display: flex;
+    text-align: center;
+    justify-content: center;
+    width: 100%;
+  } 
```

Comment:
Great work! Indentation looks good here with 2 spaces but the class names should not be indented. Please review all code and correct the indentation according to best practice at Technigo with 2 spaces.